docs/changelog.rst

```diff
@@ -6,6 +6,7 @@ A log of changes by version and date.
 ======= ========== =====
 Version Date       Notes
 ======= ========== =====
+1.28.0  09/21/2022 Added FlashBlade 2.5 and 2.6 clients
```

Comment:
Note: Manually updated this to match our actual GA date. I'll be updating this in the internal repo as well.